src/gluonts/model/inputs.py

```diff
@@ -14,15 +14,15 @@
 from collections import UserDict
 from dataclasses import dataclass
 from operator import attrgetter
-from typing import Callable, Dict, Type, Tuple
+from typing import Any, Callable, Dict, Type, Tuple
 
 from toolz import valmap
 
 
 @dataclass
 class Input:
     shape: Tuple[int, ...]
-    dtype: Type
+    dtype: Any
```

Comment:
What is the issue here?

Reply:
`torch.dtype` not recognized as a `Type` apparently

Reply:
```
src/gluonts/torch/model/deepar/module.py:161: error: Argument "dtype" to "Input" has incompatible type "dtype"; expected "Type[Any]"  [arg-type]
```